getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Get started with VPP networking
```

Comment:
Inconsistent use of bullet items...don't use periods at the end of a bullet item.